src/main/java/io/github/opencubicchunks/cubicchunks/mixin/core/common/world/level/MixinLevel.java

```diff
@@ -78,6 +79,13 @@ public CubeAccess cc_getCube(int cubeX, int cubeY, int cubeZ, ChunkStatus status
         }
     }
 
+    @Inject(method = "<init>", at = @At(value = "TAIL"))
```

Comment:
In general I'd be wary of injecting at `TAIL` instead of `RETURN` since it can miss early returns, although that's probably not relevant here.

When marking things as cubic we generally want to use `@At("io.github.opencubicchunks.cubicchunks.ConstructorSuper")` to inject as early as possible, since things during init might check whether the object is cubic.